eth-providers/src/base-provider.ts

```diff
@@ -741,8 +743,38 @@ export abstract class BaseProvider extends AbstractProvider {
 
   // Queries
   getTransaction = (transactionHash: string): Promise<TransactionResponse> => throwNotImplemented('getTransaction');
-  getTransactionReceipt = (transactionHash: string): Promise<TransactionReceipt> =>
-    throwNotImplemented('getTransactionReceipt');
+  getTransactionReceipt = async (transactionHash: string): Promise<TransactionReceipt> => {
+    const tx = await getTxReceiptByHash(transactionHash);
+
+    if (!tx) {
+      return logger.throwError(`transaction hash not found`, Logger.errors.UNKNOWN_ERROR, { transactionHash });
+    }
+
+    // TODO: correct values of these?
+    const confirmations = 0;
+    const effectiveGasPrice = BIGNUMBER_ONE;
+    const byzantium: false = false;
```

Comment:
do we need those?

I am looking at https://infura.io/docs/ethereum/json-rpc/eth-getTransactionReceipt
Also can check https://github.com/paritytech/frontier/blob/master/client/rpc-core/src/types/receipt.rs#L26

https://github.com/paritytech/frontier/blob/4dafbaf5a1a8a0f17f28511f8d9ed100750422b1/client/rpc/src/eth.rs#L1342